scripts/azureml-assets/azureml/assets/release/e2e_test.py

```diff
@@ -52,8 +58,13 @@
             data = yaml.load(fp, Loader=yaml.FullLoader)
             for test_group in data:
                 print(f"now processing test group: {test_group}")
-                p = run(f"python3 -u group_test.py -i {area} -g {test_group} -s {subscription_id} -r {resource_group} "
-                        f"-w {workspace}", shell=True)
+                cmd = f"python3 -u group_test.py -i {area} -g {test_group} -s {subscription_id} -r {resource_group} "\
+                    f"-w {workspace} -t {token}"
+                if coverage_report:
+                    cmd += f" -c {coverage_report}"
+                if version_suffix:
+                    cmd += f" -v {version_suffix}"
+                p = run(cmd, shell=True)
```

Comment:
Consider making `cmd` a list instead of a `str` to make this section simpler. See examples here: https://docs.python.org/3/library/subprocess.html#subprocess.run